### PR TITLE
network_traffic: remove invalid value from event.category in SIP sample event

### LINKED ISSUE #3329: network_traffic using invalid ECS values
```
[0] parsing field value failed: field "event.category"'s value "network_traffic" is not one of the allowed values (authentication, configuration, database, driver, email, file, host, iam, intrusion_detection, malware, network, package, process, registry, session, threat, web)
```
Part of https://github.com/elastic/integrations/issues/3016.

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

This change removes an invalid value in the SIP data set.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## How to test this PR locally

See https://github.com/elastic/integrations/issues/3016#issuecomment-1123638937

## Related issues

- Relates #3329

### CODE REVIEW

File: packages/network_traffic/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.10.1"
+  changes:
+    - description: Remove invalid value from `event.category` in SIP sample event.
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/xxxx
```
Comment (r00tu53r) [inline packages/network_traffic/changelog.yml:6]: Update PR number `xxxx` -> `3343`

Comment (efd6) [inline packages/network_traffic/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/3343
```

Comment (efd6) [reply packages/network_traffic/changelog.yml]: Thanks

**Comment by efd6** [comment]
This is an incomplete fix since the field values in question originate in packet beat in the SIP plugin. There is a change [proposed there](https://github.com/elastic/beats/pull/31599) to resolve the root issue.

**Review by r00tu53r — APPROVED** [review]

**Review by r00tu53r — APPROVED** [review]
LGTM

**Pull request merged**